docs/database/elasticsearch.md

```diff
@@ -72,13 +69,19 @@ Tracing instrumentations that do so, MUST also set `http.request.method_original
 
 **[7]:** If using a port other than the default port for this DBMS and if `server.address` is set.
 
-**[8]:** When communicating with an Elastic Cloud deployment, this should be collected from the "X-Found-Handling-Cluster" HTTP response header.
+**[8]:** If the query targets multiple indices or data streams, then the name of those should be added as a comma separated list. If the query doesn't target a specific index, this field MUST NOT be set.
 
 **[9]:** When communicating with an Elastic Cloud deployment, this should be collected from the "X-Found-Handling-Instance" HTTP response header.
 
-**[10]:** Should be collected by default for search-type queries and only if there is sanitization that excludes sensitive information.
+**[10]:** If a database system has multiple namespace components, they SHOULD be concatenated (potentially using database system specific conventions) from most general to most specific namespace component, and more specific namespaces SHOULD NOT be captured without the more general namespaces, to ensure that "startswith" queries for the more general namespaces will be valid.
```

Comment:
this note came from the generic `db.namespace` note in the registry, but it does not really make sense for elastic. 

You can override it when referencing `db.namespace` if you have anything to add there, or remove it (maybe possible with weaver).
It was not possible with older tooling and we have hacks like this one to remove the note

https://github.com/open-telemetry/semantic-conventions/blob/9381a31e07306453b38ea827f81885160ad8cada/model/trace/database.yaml#L158-L159

Reply:
Just tried: 
```yaml
        note: ""
```

works like a charm now